Bottom: 3d5acd01b190a9a227e50dfd3209e25287591137
Top:    d5cdd718f401612f76efaee51f13f28ca3fd9a3a
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-23 17:49:02 +0800

From "Lan Yixun (dlan)" <dennis.yxun@gmail.com>

autofs-5.0.7 - fix dead LDAP symbolic link when LDAP support is disabled

autofs will create symbol link mandatory no matter ldap support
is enabled or not. so, without this patch, lookup_ldaps.so will become
a dead link.


---

diff --git a/modules/Makefile b/modules/Makefile
index c5deb24..4bb1096 100644
--- a/modules/Makefile
+++ b/modules/Makefile
@@ -74,7 +74,9 @@ install: all
 	-rm -f $(INSTALLROOT)$(autofslibdir)/mount_smbfs.so
 	ln -fs lookup_file.so $(INSTALLROOT)$(autofslibdir)/lookup_files.so
 	ln -fs lookup_yp.so $(INSTALLROOT)$(autofslibdir)/lookup_nis.so
+ifeq ($(SASL), 1)
 	ln -fs lookup_ldap.so $(INSTALLROOT)$(autofslibdir)/lookup_ldaps.so
+endif
 	ln -fs mount_nfs.so $(INSTALLROOT)$(autofslibdir)/mount_nfs4.so
 ifeq ($(EXT2FS), 1)
  ifeq ($(EXT3FS), 1)
